examples/with-supertokens/app/api/user/route.ts

```diff
@@ -0,0 +1,30 @@
+import { NextResponse, NextRequest } from 'next/server';
+import {
+  getServerSideSession,
+  updateSessionInResponse,
+} from '../../components/getServerSideSession';
+
+export async function GET(request: NextRequest) {
+  const { session } = await getServerSideSession(request);
```

Comment:
why aren't we using the `withSession` function here?

Reply:
I wanted to show this method as well as `withSession` which is used elsewhere.

Reply:
I think we can just stick to recommending `withSession`. It's cleaner.